tests/pipe/test_pipe.cpp

```diff
@@ -299,22 +299,18 @@ SEQAN_DEFINE_TEST(test_pipe_test_sorter) {
     testSorter(MAX_SIZE);
 }
 
-template <typename TStringSet>
-inline void appendValues(TStringSet &stringSet, int numArgs, ...)
+template <typename TStringSet, typename T>
+inline void appendValues(TStringSet &stringSet, T t)
 {
-    va_list ap;
-    va_start(ap, numArgs);
-    for(int j = 0; j < numArgs; ++j)
-        appendValue(stringSet, va_arg(ap, char const *));
-    va_end(ap);
+    appendValue(stringSet, t);
 }
```

Comment:
```c++
va_list ap;
va_start(ap, numArgs);
for(int j = 0; j < numArgs; ++j)
    appendValue(stringSet, va_arg(ap, char const *));
va_end(ap);
```

va_list gave warnings (extension used or so) for clang.

Since it is only a convenient function in this test case, which adds a constant number of strings to a string set, I changed it to a variadic function.  For example:

```c++
appendValues(expectedOutput,
     "< 2 , [T A A] >",
     "< 2 , [T A A] >",
     "< 1 , [A A A] >");
     "< 1 , [A A A] >");
```